create.ts

```diff
@@ -0,0 +1,803 @@
+  stitches: false,
+  reactQuery: true,
+  helmet: true,
+  reactRouter: true,
+  wouter: false,
+  typescript: true,
+  style: "Twind",
+  router: "React-Router",
+  head: "",
+};
+
+/*
+INTRO CONTENT
+*/
+
+console.log(`
+▄• ▄▌▄▄▌  ▄▄▄▄▄▄▄▄   ▄▄▄· 
+█▪██▌██•  •██  ▀▄ █·▐█ ▀█ 
+█▌▐█▌██▪   ▐█.▪▐▀▀▄ ▄█▀▀█ 
+▐█▄█▌▐█▌▐▌ ▐█▌·▐█•█▌▐█ ▪▐▌
+ ▀▀▀ .▀▀▀  ▀▀▀ .▀  ▀ ▀  ▀ 
+`);
+console.log(outdent`
+    Welcome to Ultra\n
+    Let's get you setup with your new Ultra project.\n
+    But first, we need to ask just a few questions.\n
+  `);
+
+/*
+PROJECT QUESTIONS
+*/
+
+const projectName = (await question(
+  "input",
+  "What is the name of your project?",
+  "my-ultra-project",
+));
+projectName ? config.projectName = projectName : null;
+
+// const useDefault = await question(
+//   "confirm",
+//   "Do you want to use the default configuration?",
+//   true,
+// );
+
+// if (useDefault === true) {
+//   console.log("Using default configuration...");
+//   await createUltraApp();
+// }
+
+const typescript = await question(
+  "confirm",
+  "Do you want to use typescript?",
+  true,
+);
+typescript ? config.typescript = true : config.typescript = false;
+
+const style = await question("list", "Select a style library", [
+  "Twind",
+  "Emotion",
+  "Stitches",
+  "None",
+]);
+
+const router = await question("list", "Select a routing library", [
+  "React Router",
+  "Wouter",
+  "None",
+]);
+
+const head = await question("list", "Select a head management library", [
+  "React Helmet",
+  "None",
+]);
+
+const query = await question("list", "Select a query library", [
+  "React Query",
+  "None",
+]);
+
+config.emotion = style === "Emotion";
+config.stitches = style === "Stitches";
+config.twind = style === "Twind";
+config.reactRouter = router === "React Router";
+config.helmet = head === "React Helmet";
+config.reactQuery = query === "React Query";
+config.wouter = router === "Wouter";
+
+const context: TaskContext = {
+  cwd: Deno.cwd(),
+  output: config.projectName,
+  overwrite: false,
+  dialect: config.typescript ? "ts" : "js",
+};
+
+const utils: TaskUtils = {
+  dialectFilename(filename: string, isJsx?: boolean) {
+    const extension = dialectExtension(config.typescript ? "ts" : "js", isJsx);
+    return `${filename}.${extension}`;
+  },
+};
+
+async function execute(context: TaskContext, utils: TaskUtils) {
+  const tasks = taskQueue([
+    createFileTask("deno.json", denoConfigContent(utils)),
+    createFileTask("importMap.json", importMapContent()),
+    createFileTask(utils.dialectFilename("build"), buildContent()),
+    createFileTask(utils.dialectFilename("server", true), serverContent()),
+    createFileTask(utils.dialectFilename("client", true), clientContent()),
+    createFileTask(utils.dialectFilename("src/app", true), helloUltraContent()),
+    fetchFileTask(
+      "public/favicon.ico",
+      import.meta.resolve(
+        "https://github.com/exhibitionist-digital/ultra/blob/main/examples/basic/public/favicon.ico",
```

Comment:
Pretty sure you can use `import.meta.resolve('./examples/basic/public/favicon.ico')` as it will be resolved relative to where the `create.ts` script is being executed from. eg `https://deno.land/x/ultra/create.ts` would resolve to `https://deno.land/x/ultra/examples/basic/public/favicon.ico`